### PR TITLE
adding single workplace.

### PR DESCRIPTION
**Pull request by TokhirjonSoliev**

### CODE REVIEW

File: src/main/java/com/exadel/coolDesking/workspace/WorkplaceController.java [hunk 1]
```diff
@@ -42,9 +42,21 @@
...
+     * @param office_id
+     * @param floorPlan_id
+     * @param workplaceCreateDto
+     * @return status created
+     */
     @PostMapping("/floorPlan/{floorPlan_id}/workplace")
-    public ResponseEntity<?> addWorkplace(MultipartFile file, @PathVariable UUID office_id, @PathVariable UUID floorPlan_id) {
-        workplaceService.addWorkplace(file, office_id, floorPlan_id);
+    public ResponseEntity<?> addWorkplace(@PathVariable UUID office_id, @PathVariable UUID floorPlan_id, @RequestBody @Valid WorkplaceCreateDto workplaceCreateDto){
+        workplaceService.addWorkplace(office_id, floorPlan_id, workplaceCreateDto);
```
Comment (gargatuma) [inline src/main/java/com/exadel/coolDesking/workspace/WorkplaceController.java:59]: office_id, floorPlan_id should be changed to officeId, floorPlanId

File: src/main/java/com/exadel/coolDesking/workspace/WorkplaceController.java [hunk 2]
```diff
@@ -42,9 +42,21 @@
      * @param file, office_id ,workplace_id
      * @return status created
      */
+    @PostMapping("/floorPlan/{floorPlan_id}/workplace/bulk")
+    public ResponseEntity<?> addWorkplaces(MultipartFile file, @PathVariable UUID office_id, @PathVariable UUID floorPlan_id) {
```
Comment (gargatuma) [inline src/main/java/com/exadel/coolDesking/workspace/WorkplaceController.java:46]: office_id, floorPlan_id should be changed to officeId, floorPlanId

**Review by gargatuma — CHANGES_REQUESTED** [review]

File: src/main/java/com/exadel/coolDesking/workspace/WorkplaceController.java (hunk 1 above)
Comment (TokhirjonSoliev) [reply src/main/java/com/exadel/coolDesking/workspace/WorkplaceController.java:59]: ah, I forgot to change.

Comment (TokhirjonSoliev) [reply src/main/java/com/exadel/coolDesking/workspace/WorkplaceController.java:59]: I think, they will not changed here

**Pull request closed without merging**